src/tennisSimulator/MatchSimulator.java

```diff
@@ -0,0 +1,30 @@
+package tennisSimulator;
+
+public class MatchSimulator {
+
+    private final Player playerOne;
+    private final Player playerTwo;
```

Comment:
Now you have the same concept duplicated in two classes here, and in MatchParticipants. If you decided to have player fields here then probably MatchParticipants is not needed, I don't see value to have both of them, since MatchParticipants now is used only in this class constructor for initialization